core/ui/navigation/routes.tsx

```
@@ -1,13 +1,20 @@
 import { ReshareVaultPage } from '@core/ui/mpc/keygen/reshare/ReshareVaultPage'
 import { CorePath } from '@core/ui/navigation'
 import { CurrencyPage } from '@core/ui/preferences/currency'
+import { VaultDefaultChainsPage } from '@core/ui/preferences/vaultDefaultChains/VaultDefaultChainsPage'
 import { ActiveVaultGuard } from '@core/ui/vault/ActiveVaultGuard'
 import { ReactNode } from 'react'
 
-type SharedPaths = Extract<CorePath, 'currencySettings' | 'reshareVault'>
+import { ReshareVaultPage } from '../mpc/keygen/reshare/ReshareVaultPage'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ReshareVaultPage'. Consider to delete it or rename it.

'ReshareVaultPage' is defined here:


(lint/suspicious/noRedeclare)